content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -100,24 +100,22 @@ If you are on macOS and using [Macports](https://macports.org/) package manager,
 If you are on Windows and using [Powershell Gallery](https://www.powershellgallery.com/) package manager, you can install and update kubectl with Powershell.
 
 To install:
-* Run the installation commands (making sure to specify a DownloadLocation):
+* Run the installation commands (making sure to specify a `DownloadLocation`):
 
     ```
     Install-Script -Name install-kubectl -Scope CurrentUser -Force
     install-kubectl.ps1 [-DownloadLocation <path>]
     ```
     
     {{< note >}}
-    **Note:** If you do not specify a DownloadLocation, kubectl will be installed in the user's temp Directory.
+    **Note:** If you do not specify a `DownloadLocation`, `kubectl` will be installed in the user's temp Directory.
     {{< /note >}}
-    The installer creates $HOME/.kube and instructs it to create a config file
+    The installer creates `$HOME/.kube` and instructs it to create a config file
+
 To update:
 * Run the update commands:
```

Comment:
Please update this one as well:  Please update the * to instead follow the structure shown in the ## Install with Chocolatey on Windows section below (using numbers instead).  The * is not rendering properly